.github/workflows/build-and-test.yml

```diff
@@ -182,12 +182,58 @@ jobs:
         uses: ./.github/actions/docker-run-playwright
         with:
           entrypoint: run-test-vrt-playwright
-          args: --components-js
+          args: --components-js:chrome:common
+      - name: 'VRT Regression (Playwright)'
+        if: failure()
+        uses: actions/upload-artifact@v3
+        with:
+          name: components-js-common-vrt-playwright-chrome-regression
+          path: ./packages/components-js/tests/vrt/playwright/results
+  components-js-custom-states-vrt-playwright-chrome:
+    name: 'Components JS Custom/States VRT (Playwright Chrome)'
```

Comment:
Maybe we should stick with `Components JS VRT ..` for consistent naming.
<img width="320" alt="Bildschirmfoto 2023-11-20 um 09 55 19" src="https://github.com/porsche-design-system/porsche-design-system/assets/4380295/94e087ae-a6a7-454d-b41e-5dc0dbeda7af">